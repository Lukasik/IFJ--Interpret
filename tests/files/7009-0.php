<?php

function a($x)
{
	$x = b($x);
	return $x;
}

function b($x)
{
	$x = c($x);
	return $x;
}

function c($x)
{
	while($x)
	{
		$x = $x - 1;
		$z = put_string($x);
	}

	return 42;
}

$x = a(9);
$z = put_string($x);

$x = c(9);
