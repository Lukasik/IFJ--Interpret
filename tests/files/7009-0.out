87654321042876543210